### ISSUE TITLE
Is there a implement of Subvector-wise Clustering?

### ISSUE THREAD

**Issue by lovelavez** [ArnavS11/visual-search-engine#1]
The paper compared the Element-wise Rounding and Subvector-wise Clustering in Figure.4. And it seems  Subvector-wise Clustering is the point, Element-wise Rounding was just used to make comparison. But i didn't find out how to test Element-wise Rounding in this repositoriy. Is there one?

**Comment by ArnavS11** [comment]
Yeah you are correct, unfortunately we didn't implement it with the proposed subvector-wise clustering, as we were required in the project to just provide any kind of working implementation for this usecase. I'll add a note to clear this up for anyone else in the future. Thanks for pointing it out.

**Comment by ArnavS11** [comment]
Yeah you are correct, unfortunately there is no implementation here of subvector-wise clustering as proposed in the paper as we were just required to submit any working implementation for the usecase. I'll write a note to clarify for anyone else viewing in the future. Thanks for pointing it out I had forgotten mentioning.
Also you want to test Element-wise rounding? There is a module for that as far as I can recall. But if you meant subvector-wise clustering then not. 
Also, feel free to contribute if you happen to implement that.
